setup/nextcloud.sh

```diff
@@ -21,26 +21,29 @@ echo "Installing Nextcloud (contacts/calendar)..."
 #   we automatically install intermediate versions as needed.
 # * The hash is the SHA1 hash of the ZIP package, which you can find by just running this script and
 #   copying it from the error message when it doesn't match what is below.
-nextcloud_ver=25.0.7
-nextcloud_hash=a5a565c916355005c7b408dd41a1e53505e1a080
+nextcloud_ver=26.0.7
+nextcloud_hash=ead5a81b0ddc145220dcabea3aa32664b78b7aa6
 
 # Nextcloud apps
 # --------------
-# * Find the most recent tag that is compatible with the Nextcloud version above by
-#   consulting the <dependencies>...<nextcloud> node at:
-#   https://github.com/nextcloud-releases/contacts/blob/main/appinfo/info.xml
```

Comment:
These links even point to untested alpha releases, so I've changed them to official, tested releases

Reply:
That's for contacts and calendar. user_external is an exception :(